plugins/modules/nd_compliance_requirement_config_manual.py

```diff
@@ -0,0 +1,340 @@
+author:
+- Akini Ross (@akinross)
+options:
+  insights_group:
+    description:
+    - The name of the insights group.
+    type: str
+    required: yes
+    aliases: [ fab_name, ig_name ]
+  name:
+    description:
+    - The name of the compliance requirement.
+    type: str
+  description:
+    description:
+    - The description of the compliance requirement.
+    type: str
+    aliases: [ descr ]
+  enabled:
+    description:
+    - Enable the compliance requirement.
+    type: bool
+  sites:
+    description:
+    - Names of the Assurance Entities.
+    type: list
+    elements: str
+  object:
+    description:
+    - Container for all matching criteria attached to the object.
+    type: dict
+    suboptions:
+      type:
+        description:
+        - The object type of the object.
+        type: str
+        required: yes
+        choices: [ tenant, vrf, bd, epg, contract, subject, filter ]
+      includes:
+        description:
+        - Container for all matching criteria to include.
+        type: list
+        required: yes
+        elements: dict
+        suboptions:
+          type:
+            description:
+            - The object type of the match.
+            type: str
+            required: yes
+            choices: [ tenant, vrf, bd, epg, ap, l3out, l3instp, l2out, l2instp, filter, subject, contract ]
+          attribute:
+            description:
+            - The attribute of the match.
+            - The GUI represent this as 'By'.
+            type: str
+            choices: [ DN ]
+            default: DN
+          patterns:
+            description:
+            - Container for all patterns attached to the match.
+            type: list
+            required: yes
+            elements: dict
+            suboptions:
+              type:
+                description:
+                - The type of the match.
+                type: str
+                required: yes
+                choices: [ tenant, vrf, bd, epg, ap, l3out, l3instp, l2out, l2instp, filter, subject, contract ]
+              operator:
+                description:
+                - The operator of the pattern.
+                type: str
+                required: yes
+                choices: [ contains, begins_with, ends_with, equal_to, not_equal_to, not_contains, not_begins_with, not_ends_with ]
+              value:
+                description:
+                - The value of the pattern to match on.
+                - Not providing a pattern sets to ANY-STRING.
+                type: str
+      excludes:
+        description:
+        - Container for all matching criteria to exclude.
+        type: list
+        elements: dict
+        suboptions:
+          type:
+            description:
+            - The object type of the match.
+            type: str
+            required: yes
+            choices: [ tenant, vrf, bd, epg, ap, l3out, l3instp, l2out, l2instp, filter, subject, contract ]
```

Comment:
are the external epgs (l2instp and l3instp) referenced as External EPGs or instps in the UI?